testutils/random_string.go

```diff
@@ -0,0 +1,21 @@
+package testutils
```

Comment:
This looks good to me. Could you also move the `testlogr.go` into this package and update the references throughout the project? It would be good to minimize the number of test related packages we create.

Reply:
Sure!